docs/guide/content_rendering/queries_and_controllers/controllers.md

```diff
@@ -0,0 +1,26 @@
+# Controllers
+
+Configuring a controller enables you to modify and enhance the way in which the built-in content view controller renders content.
+
+You indicate which controller to use in the [content view configuration](../templates/template_configuration.md), under the `controller` key:
+
+``` yaml
+[[= include_file('code_samples/front/embed_content/config/packages/views.yaml', 23, 26) =]][[= include_file('code_samples/front/embed_content/config/packages/views.yaml', 28, 30) =]]
+```
+
+A custom controller must extend `eZ\Bundle\EzPublishCoreBundle\Controller`.
```

Comment:
Is this really true?

Reply:
Ah, I was hoping you'd know ;)

Reply:
I think it is not needed but let me check then :D

Reply:
@DominikaK 
I now can confirm, that this controller does not have to extend anything in order to this to work.